cmd/jaeger/docs/migration/all-in-one-metrics.md

```diff
@@ -0,0 +1,92 @@
+### Common Metrics
```

Comment:
could we just have a single table with a format similar to `### Equivalent Metrics`? When one side is missing, put n/a